packages/web3-eth/src/index.ts

```diff
@@ -37,20 +85,23 @@ export default class Web3Eth {
      * @returns {Promise} Client version
```

Comment:
Comments need to be updated from rpcOptions to CallOptions

Reply:
Going to update the doc blocks as part of the `web3-eth` type updates

Reply:
Fix by #4114